src/components/fields/date-field/types.ts

```diff
@@ -11,11 +12,12 @@ export interface IDateFieldValidationRule extends IYupValidationRule {
 	minDate?: string | undefined;
 	maxDate?: string | undefined;
 	excludedDates?: string[] | undefined;
+	withinDays?: IWithinDaysRule;
 }
 
 export interface IDateFieldSchema<V = undefined>
 	extends IBaseFieldSchema<"date-field", V, IDateFieldValidationRule>,
-		TComponentOmitProps<DateInputProps, "minDate" | "maxDate" | "disabledDates"> {
+		TComponentOmitProps<DateInputProps, "minDate" | "maxDate" | "disabledDates" | "withinDays"> {
```

Comment:
not needed, we use `TComponentOmitProps` to exclude props from DS interfaces but `withinDays` do not exist there in the first place